### PR TITLE
Imports and top level variables

### PR DESCRIPTION
**Pull request by jrc2139**
I noticed installing with `pub get` didn't pull the latest code from github.

### CODE REVIEW

**Comment by jrc2139** [comment]
included new tags

**Comment by jrc2139** [comment]
this will install `dart_ctags` as a bin

File: bin/tags.dart [hunk 1]
```diff
@@ -174,12 +197,14 @@ class Ctags {
...
-              'C',
+              'r',
+              'access:${name[0] == '_' ? 'private' : 'public'}',
               options['line-numbers'] as bool
                   ? 'line:${unit.lineInfo.getLocation(offset).lineNumber}'
                   : '',
               'class:${declaration.name}',
-              'signature:(${_parseArgs(member.toSource())})',
+              'signature:${member.parameters.toString()}',
+              // 'signature:(${_parseArgs(member.toSource())})',
```
Comment (nerdrew) [inline bin/tags.dart]: leftover commented out code?

Comment (jrc2139) [reply bin/tags.dart]: good catch!

**Comment by nerdrew** [comment]
Is this ready to merge? Do you want to squash commits or do you prefer the history?

**Comment by jrc2139** [comment]
yeah, it's been working really well for me, and I'm really curious to see how it installs from pub now.  I'm coming back to it every now and then, and trying to match it as closely as possible to the dart api docs.

**Comment by nerdrew** [comment]
Published. Want to give it a test?

**Comment by nerdrew** [comment]
I get errors when I open tagbar. Is there a specific tagbar config needed to make this work?

```
Error detected while processing function <SNR>141_AutoUpdate[63]..<SNR>141_ProcessFile[121]..<SNR>165_sortTags[2]..tagbar#sorting#sort[6]..<SNR>169_compare_by_kind:                                                                         
line    3:
E716: Key not present in Dictionary: F
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: F
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: F
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: r
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: F
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: r
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: r
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: F
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
E716: Key not present in Dictionary: F
E15: Invalid expression: typeinfo.kinddict[a:tag1.fields.kind] <# typeinfo.kinddict[a:tag2.fields.kind]
```

**Comment by nerdrew** [comment]
FWIW I use neovim, though I assume tagbar works the same there. Crap. JK. I probably have a custom dart config. Let me check.

**Comment by nerdrew** [comment]
Hmmm. No custom dart config. I needed

```
let g:tagbar_type_dart = { 'ctagsbin': '~/.pub-cache/bin/dart_ctags' }
```

Should we add that to the README?

**Comment by nerdrew** [comment]
Seems to work. Thanks for the contributions!!!

**Comment by nerdrew** [comment]
Look right?

```
+### Use with Tagbar (the vim plugin)
+
+To use with [tagbar](https://github.com/majutsushi/tagbar), add this to your vim config:
+
+```vimscript
+let g:tagbar_type_dart = { 'ctagsbin': '~/.pub-cache/bin/dart_ctags' }
+```
+
```

**Comment by jrc2139** [comment]
> Look right?
> 
> ```
> +### Use with Tagbar (the vim plugin)
> +
> +To use with [tagbar](https://github.com/majutsushi/tagbar), add this to your vim config:
> +
> +```vimscript
> +let g:tagbar_type_dart = { 'ctagsbin': '~/.pub-cache/bin/dart_ctags' }
> +```
> +
> ```

okay I think it's because if you don't do the `dart2native` step, then the `execute` function of vim sees that the `dart_ctags` is just a bash script, so then it will fail.

**Comment by nerdrew** [comment]
It works for me either way: both the bash wrapper and the `dart2native` version work with the tagbar `ctagsbin` setting above.

**Comment by jrc2139** [comment]
> It works for me either way: both the bash wrapper and the `dart2native` version work with the tagbar `ctagsbin` setting above.

right but I fixed [tagbar specifically](https://github.com/majutsushi/tagbar/pull/581) to detect the executable.  then no configuration is needed for tag bar.

**Comment by nerdrew** [comment]
ah, I don't have `~/.pub-cache/bin` on my path. That's why it didn't work for me.

**Comment by nerdrew** [comment]
Updated the readme.

**Pull request merged**